lib/stellar_core_commander/transactor.rb

```diff
@@ -356,6 +356,15 @@ def set_upgrades(protocolversion)
       @process.set_upgrades protocolversion
     end
 
+    Contract None => Any
+    def check_uses_latest_protocol_version
+      latest_protocol_version = @process.info["protocol_version"]
```

Comment:
when https://github.com/stellar/stellar_core_commander/pull/119 gets merged, this can be removed and call to `protocol_version` can be used instead.

Reply:
I'll do that now.